content/realtime/channel-params.textile

```diff
@@ -0,0 +1,158 @@
+---
+title: Channel Parameters
+section: realtime
+index: 32
+jump_to:
+  Help with:
+    - Overview#overview
+    - Using params with v1.2+ Ably libraries#using-params-with-lib-v12
+    - Using params with v1.1 or earlier Ably libraries#using-params-with-lib-v11
+    - Using params with non-Ably transports#using-params-with-other-transports
+    - Examples#examples
+  Parameters:
+    - rewind#rewind
+---
+
+h2(#overview). Overview
+
+Channel parameters are a general mechanism by which a client can express properties of a channel, or of its attachment to a channel. A set of channel parameters is simply a set of key/value pairs, where both keys and values are strings; the keys correspond to specific features that Ably defines.
+
+The methods provided for specifying channel parameters, and the currently available features, are outlined below.
+
+h2(#supported-params). Currently supported channel params
+
+- rewind := Used to request that an attachment start from some number of messages or point in time in the past. See "rewind":#rewind for more information
+- delta := **In beta**. Used to request that data payloads should be sent as deltas to the previous payload. "Contact us":https://www.ably.io/contact for more information and supported values
```

Comment:
I don't think we should be saying that deltas are in beta. Beta implies that it's feature-complete, and it isn't yet (because the library functionality isn't released yet). I think we should be describing this as experimental for now.